### PR TITLE
Fix checking if scheduler has a replica assigned to each cluster

### PR DESCRIPTION
**Pull request by AllenZMC**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Fix checking if scheduler has a replica assigned to each cluster
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by RainbowMango** [comment]
Can you explain how to reproduce this bug?

**Comment by AllenZMC** [comment]
@RainbowMango  There is no such problem yet, but logically this code has loopholes, look at my test case.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Hi @AllenZMC, the comment of this method may mislead you. If the number of replicas of a cluster is greater than 0, it indicates that the scheduler has allocated replicas to these clusters, even if the number of replicas of some clusters is 0.

**Comment by AllenZMC** [comment]
@XiShanYongYe-Chang Thank you for the explanation. That is to say, as long as the number of replicas in a cluster is greater than 0, it indicates that the cluster has been scheduled?

**Comment by XiShanYongYe-Chang** [comment]
> @XiShanYongYe-Chang Thanks for the explanation. That is to say, as long as the number of replicas in a cluster is greater than 0, it indicates that the cluster has been scheduled?
> 
> We need to change the description of the comment, it really misleads me.

Yes, you can update the comment and add the ut for the test.

**Comment by XiShanYongYe-Chang** [comment]
Hi @AllenZMC, maybe you need to rebase the master branch and push it again.

**Comment by AllenZMC** [comment]
@XiShanYongYe-Chang  rebased.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks!

/lgtm

File: pkg/util/helper/binding_test.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+)
+
+func TestHasScheduledReplica(t *testing.T) {
+	tests := []struct {
+		name           string
+		scheduleResult []workv1alpha2.TargetCluster
+		want           bool
+	}{
+		{
+			name: "assigned replicas for a cluster",
```
Comment (RainbowMango) [inline pkg/util/helper/binding_test.go:16]: The name of sub-test should keep unique for distinguishing them from each other.

Comment (AllenZMC) [reply pkg/util/helper/binding_test.go]: Fixed. @RainbowMango

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**